### ISSUE TITLE
[22.5.0] Code signatur broken

### ISSUE THREAD

**Issue by tuxmaster5000** [greenbone/openvas-smb#56]
The file for the code signature is broken. GPG will reject this file for the tar ball.

**Comment by nichtsfrei** [comment]
Hi, yes, that was an oversight on my part, I uploaded new signatures. Can you try them?

**Comment by tuxmaster5000** [comment]
Yes now it will work. But all other 22.5.0 releases are also affected. (openvas-scanner, notus etc.)

**Comment by nichtsfrei** [comment]
That's true, but they should also have new signatures by now.

**Comment by tuxmaster5000** [comment]
Thanks
